lib/screens/v2/widget_instance/reconstructed_alert.ex

```diff
@@ -796,77 +796,44 @@ defmodule Screens.V2.WidgetInstance.ReconstructedAlert do
     }
   end
 
-  defp boundary_flex_fields(%__MODULE__{alert: %Alert{effect: :suspension}} = t, location) do
-    %__MODULE__{alert: %{cause: cause, header: header}} = t
-    affected_routes = LocalizedAlert.consolidated_informed_subway_routes(t)
-
-    if length(affected_routes) > 1 do
-      %{
-        issue: header,
-        remedy: "Seek alternate route",
-        location: "",
-        cause: "",
-        routes: get_route_pills(t),
-        effect: :suspension,
-        urgent: true
-      }
-    else
-      destination = get_destination(t, location)
-      cause_text = Alert.get_cause_string(cause)
-      issue = if is_nil(destination), do: "No trains", else: "No #{destination} trains"
-
-      %{
-        issue: issue,
-        remedy: "Seek alternate route",
-        location: "",
-        cause: cause_text,
-        routes: get_route_pills(t),
-        effect: :suspension,
-        urgent: true
-      }
-    end
-  end
+  # Boundary presentation of shuttle/suspension alerts is also valid when inside them.
+  defp inside_flex_fields(%__MODULE__{alert: %Alert{effect: effect}} = t)
+       when effect in ~w[shuttle suspension]a,
+       do: boundary_flex_fields(t, :inside)
 
-  defp boundary_flex_fields(%__MODULE__{alert: %Alert{effect: :shuttle}} = t, location) do
+  defp boundary_flex_fields(%__MODULE__{alert: %Alert{effect: effect}} = t, location)
+       when effect in ~w[shuttle suspension]a do
     %__MODULE__{alert: %{cause: cause, header: header}} = t
     affected_routes = LocalizedAlert.consolidated_informed_subway_routes(t)
+    remedy = if(effect == :shuttle, do: "Use shuttle bus", else: "Seek alternate route")
 
     if length(affected_routes) > 1 do
       %{
         issue: header,
-        remedy: "Use shuttle bus",
+        remedy: remedy,
         location: "",
         cause: "",
         routes: get_route_pills(t),
-        effect: :shuttle,
+        effect: effect,
         urgent: true
       }
     else
       destination = get_destination(t, location)
       cause_text = Alert.get_cause_string(cause)
-
-      issue =
-        if is_nil(destination) do
-          "No trains"
-        else
-          "No #{destination} trains"
-        end
+      issue = if is_nil(destination), do: "No trains", else: "No #{destination} trains"
 
       %{
         issue: issue,
-        remedy: "Use shuttle bus",
+        remedy: remedy,
         location: "",
         cause: cause_text,
         routes: get_route_pills(t),
-        effect: :shuttle,
+        effect: effect,
         urgent: true
       }
     end
   end
 
-  defp boundary_flex_fields(%__MODULE__{alert: %Alert{effect: :station_closure}}, _location),
-    do: nil
-
```

Comment:
It doesn't look like we have a case to handle `%Alert{effect: :station_closure}` anymore. Is that intentional?

Reply:
gotcha, makes sense! I could see us having a comment explaining that or Log a warning if we do end up in that case, but I don't think that's needed before merging this fix